p4-16/spec/P4-16-draft-spec.mdk

```diff
@@ -2636,9 +2634,9 @@ error errorFromParser;
 if (errorFromParser != error.NoError) { ... }
 ~ End P4Example
 
-## Expressions on enum values { #sec-enum-exprs }
+## Operations on expressions of type ```enum``` { #sec-enum-exprs }
 
-Symbolic names declared by an ```enum``` do not belong to the
+Symbolic names declared by an ```enum``` declaration do not belong to the
 top-level namespace, but to a newly introduced namespace.
```

Comment:
Symbolic names declared by an ```enum``` belong to the namespace introduced by the enum declaration rather than the top-level namespace.